src/Symfony/Component/Mailer/DelayedSmtpEnvelope.php

```diff
@@ -73,7 +73,7 @@ private static function getRecipientsFromHeaders(Headers $headers): array
         foreach (['to', 'cc', 'bcc'] as $name) {
             foreach ($headers->all($name) as $header) {
                 foreach ($header->getAddresses() as $address) {
```

Comment:
Is it possible to drop one nested loop by using `$recipients = array_merge($recipients, $header->getAddresses()`?